nvflare/fuel/hci/server/file_transfer.py

```diff
@@ -31,6 +32,7 @@
 from nvflare.fuel.hci.conn import Connection
 from nvflare.fuel.hci.reg import CommandModule, CommandModuleSpec, CommandSpec
 from nvflare.fuel.hci.zip_utils import unzip_all_from_bytes, zip_directory_to_bytes
+from nvflare.private.fed.server.server_engine import ServerEngine
```

Comment:
I think a check to make sure engine is ServerEngine is important as that is the way we ensure engine.job_def_manager exists.

But I think the "fuel" package should not import class from "private"?

Reply:
removed the imports for now

Reply:
If the file_transfer module depends on server_engine's job_def_manager instance,
should we move the file_transfer module into "private"?